x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/load_global_execution_kpi_aggregations.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import { IExecutionKPIResult } from '@kbn/alerting-plugin/common';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../../constants';
+
+export interface LoadGlobalExecutionKPIAggregationsProps {
+  id: string;
+  outcomeFilter?: string[];
+  message?: string;
+  dateStart: string;
+  dateEnd?: string;
+}
+
+const getFilter = ({ outcomeFilter, message }: { outcomeFilter?: string[]; message?: string }) => {
```

Comment:
nit: could we extract this `getFilter` function for reuse? Seems like it's repeated in 4 places now

Reply:
We will take care of if later on -> https://github.com/elastic/kibana/issues/141833

Reply:
fix with that 093c0dc3b4d9b0f03120113849728c667a38344a